pdr_backend/aimodel/test/test_aimodel_prices.py

```diff
@@ -165,7 +165,7 @@ def _run_one_iter(autoregr_n, max_n_train, transform, sim_test_n, test_i) -> flo
     X_train, X_test = X[st_:fin, :], X[fin : fin + 1, :]
     ytran_train, _ = ytran[st_:fin], ytran[fin : fin + 1]
 
-    curprice = ysignal[-2]
```

Comment:
Don't change the names in this file. Why: this file is more general, it doesn't matter, let's keep the PR small.